packages/aws-cdk-lib/aws-rds/lib/instance.ts

```diff
@@ -1204,6 +1218,34 @@ export class DatabaseInstance extends DatabaseInstanceSource implements IDatabas
 
     this.setLogRetention();
   }
+
+  /**
+   * Determines the value of the `storageEncrypted` property for the instance.
+   */
+  private getStorageEncryptedProperty(props: DatabaseInstanceProps): boolean | undefined {
```

Comment:
What about creating a shared method in `/private` for both the instance and the cluster?
```ts
export const getStorageEncryptedProperty(storageEncryptionKey?: boolean, storageEncryptedLegacyDefaultValue?: boolean): boolean | undefined { ... }
```

Reply:
Great idea! [`3c317b0` (#32695)](https://github.com/aws/aws-cdk/pull/32695/commits/3c317b0e6d8bf43a22fce47d81ea7e391a9e3305)